dataikuapi/dss/modelevaluationstore.py

```diff
@@ -288,6 +289,31 @@ def delete(self):
         self.client._perform_json(
                 "DELETE", "/projects/%s/modelevaluationstores/%s/runs/" % (self.project_key, self.mes_id), body=obj)
 
+    @property
+    def full_model_like_id(self):
+        return "ME-%s-%s-%s"%(self.project_key, self.mes_id, self.run_id)
+
+    def compute_data_drift(self, reference=None, data_drift_params=None):
+        """
+        Compute data drift against a reference model or model evaluation. The reference is determined automatically unless specified.
+
+        :param reference: saved model version or model evaluation to use as reference (optional, accepts ID or Python object)
+        :type reference: Union[str, DSSModelEvaluation, DSSTrainedPredictionModelDetails]
+        :param data_drift_params: data drift computation settings (optional)
+        :return: data drift analysis results, as a JSON object
```

Comment:
Sorry, I have a last remark. A DSSFuture is returned, not a JSON object. I would write something like:
```
:returns: a :class:`dataikuapi.dss.future.DSSFuture` tracking the progress of the update. Call 
                   :meth:`~dataikuapi.dss.future.DSSFuture.wait_for_result` on the returned object
                   to wait for completion (or failure) and get the result as a JSON object
```

Reply:
Handled differently (auto wait by default, much more practical)